install/docker/console-db-es-ls-kibana/docker-compose.yml

```diff
@@ -121,7 +121,7 @@ services:
       retries: 12
     volumes:
       - ./config.properties:/home/cws_user/config.properties:ro
-      - logs-volume:/home/cws_user/cws/server/apache-tomcat-9.0.58/logs
+      - console-logs-volume:/home/cws_user/cws/server/apache-tomcat-9.0.33/logs
```

Comment:
@brentjk Since the original base branch of the pull request was `main`, the Apache Tomcat version was reverted to 9.0.33. The `develop` branch is currently using Tomcat version 9.0.58, the version used by Camunda Platform 7.18. So, before merging the pull request to `develop,` the Tomcat version will have to be updated.

Reply:
fixed